manager/test/api/src/test/resources/test-plan-data/apiman-2471/001_import.resttest

```diff
@@ -0,0 +1,315 @@
+POST /system/import admin/admin
+Content-Type: application/json
+
+{
+   "Metadata": {
+     "id": 1686577330281,
+     "apimanVersion": "8.6.0-SNAPSHOT"

```

Comment:
I take it this is your downstream version!

Reply:
Well spotted, created the test data with that version.
Can change it if you want, but it should not affect any test runs